domain-management/src/main/java/org/jboss/as/domain/management/security/UserLdapCallbackHandler.java

```diff
@@ -88,7 +88,7 @@
 
     private final boolean allowEmptyPassword;
     private final boolean shareConnection;
-    protected final int searchTimeLimit = 10000; // TODO - Maybe make configurable.
+    protected static final int searchTimeLimit = 10000; // TODO - Maybe make configurable.
```

Comment:
as far as I can tell, this field is not used at all. I'm not sure we need to make it static as it's not clear what its usage would be.

@darranl could you shed some light on it? Could we delete it completely?

Reply:
I am not going to be looking at this, at the moment my tasks are to remove legacy security.